exercises/practice/list-ops/.docs/instructions.append.md

```diff
@@ -0,0 +1,101 @@
+# placeholder
+
+## New AWK concepts
+
+There are several interesting new concepts we're seeing in this exercise.
+
+### Include files
+
+This is the first exercise we've seen where the solution we're writing is
+not a "main" script. We're writing a library to be included into other awk
+programs.  You'll see the [`@include`][d-include] directive in the test
+file.  This is analagous to the shell's `source` (aka `.`) command.
```

Comment:
If you're going to describe what `@include` does, consider describing it without just referencing another language.